apps/dhparam.c

```diff
@@ -55,6 +55,7 @@ const OPTIONS dhparam_options[] = {
     OPT_R_OPTIONS,
     {"C", OPT_C, '-', "Print C code"},
     {"2", OPT_2, '-', "Generate parameters using 2 as the generator value"},
+    {"3", OPT_3, '-', "Generate parameters using 3 as the generator value"},
```

Comment:
That seems to be a new feature.

Reply:
Ah, sure, I can remove that part.
is it okay to keep the GENERATOR_3 define?